backend/src/contributors.ts

```diff
@@ -38,6 +38,21 @@ export class Contributors {
         return _.sortBy(collections, ['type']);
     }
 
+    public getCollectionsInfo(): Array<any> {
+        const collections: Array<any> = [];
+        for (const extensionId of this.collectionsMap.keys()) {
+            this.collectionsMap.get(extensionId).forEach(collection => {
+                collections.push({
+                    extensionId: extensionId,
```

Comment:
A tiny style about this property: it could also be a shorthand property like this `{ extensionId }`?

Reply:
Yes, "extensionId" may also be a shorthand property here.  Due to different values and names of other attributes，so I didn't use shorthand attributes for consistency.